Makefile

```diff
@@ -25,8 +25,12 @@ GRPC_DIR:=./vendor/google.golang.org/grpc
 install: submodule
 	go install $(PACKAGES)
 
+# Generate certs for unit and release tests.
+certs:
+	./cert-gen
+
 # Local test
-test: submodule
+test: submodule certs
```

Comment:
to fix the coverage build and in general : 

let's rename submodule->dependencies in every dependency clause

then make

dependencies: submodule certs

(both submodule and dependencies should be .PHONY

CERTS:=ca.crt server.crt server.key

certs: $(CERTS)

and confirm that the second time it doesn't redo the generation

Reply:
ack